packaging/homebrew-arm64/bee.yaml

```diff
@@ -40,8 +40,8 @@ data-dir: /opt/homebrew/var/lib/swarm-bee
 password-file: /opt/homebrew/var/lib/swarm-bee/password
```

Comment:
@martinconic I noticed that the homebrew-arm64/bee.yaml file seems to use different quotation styles compared to the others. Should we standardize it for consistency?

Reply:
good catch, thanks